Nick & Angela

nklunder opened the pull request:
We used skeleton grid system, flexbox and icon-fonts for top-navigation, and used CSS calc for centering the header on the page.

nklunder committed:
add original codebase

AngelaDiaFazio committed:
bleh

nklunder committed:
move code into skeleton grid, remove broken CSS

nklunder committed:
add CSS calc to center site logo

nklunder committed:
add logo image to header, remove a few unused class attributes from HTML

nklunder committed:
add flexbox navigation and style navigation links

AngelaDiaFazio committed:
new html

nklunder committed:
add extra sections to page

nklunder committed:
add icon fonts

nklunder committed:
add icon fonts, again

litteredwitherrors commented on Nick-Angela/_previous-assignment/index.html:
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+
```
Why space here?

litteredwitherrors commented on Nick-Angela/_previous-assignment/index.html:
```diff
@@ -0,0 +1,60 @@
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel ="stylesheet" href ="style2.css">
```
Remove the spaces after the attributes.

litteredwitherrors commented on Nick-Angela/_previous-assignment/index.html:
```diff
@@ -0,0 +1,60 @@
+    <body>
+      <header role ="banner">
+        <h1>Pitchfork</h1>
+        <nav role ="navigation"><h1 class="hidden">mainNav</h1>
```
Again with the spaces. hyphens instead of camel casing.

litteredwitherrors commented on Nick-Angela/_previous-assignment/style_new.css:
```diff
@@ -0,0 +1,137 @@
+/* Base */
+
```
Too many empty lines

litteredwitherrors commented on Nick-Angela/_previous-assignment/style_new.css:
```diff
@@ -0,0 +1,137 @@
+
+.otherArticle__author {
+  font-size: 1.6em;
+
```
More empty lines.

litteredwitherrors commented on Nick-Angela/index.html:
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
```
Too many indentssss everrryyy wheeerrreee.

litteredwitherrors replied:
Still a problem.

litteredwitherrors commented:
I need you guys to clean up the entire project's formatting (Indentation, white spaces, etc.)

nklunder committed:
fix things mentioned in comments, remove previous assignment folder left in

nklunder commented:
Changes made.  I apologize, we accidentally left the folder with the previous assignment's code in the repo, which is where most of the comments are from

nklunder committed:
add double-size feature-image for retina displays, convert px to em

AngelaDiaFazio committed:
px to em conversion

nklunder committed:
fix mobile media query for display of navbar

nklunder commented:
changes for friday's assignment:
added 2x scale image for retina displays,
used media query to display nav correctly at mobile size,
converted px to relative units

nklunder committed:
fix index.html formatting
